core/src/main/java/co/aikar/commands/BaseCommand.java

```diff
@@ -64,69 +65,228 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+/**
+ * BaseCommand stores data about a single command tree.
+ * This includes {@link CommandManager who} manages it, what {@link RegisteredCommand subcommands} it has and so on.
+ */
 @SuppressWarnings("unused")
 public abstract class BaseCommand {
-
+    /**
+     * This is a field which contains the key in the {@link #subCommands} map for the method to catch any unknown
+     * argument to command states.
+     */
     public static final String CATCHUNKNOWN = "__catchunknown";
+
+    /**
+     * This is a field which contains the key in the {@link #subCommands} map for the method which is default for the
+     * entire (sub)command.
+     */
     public static final String DEFAULT = "__default";
+
+    /**
+     * Stores all the subcommands of this command.
```

Comment:
same as below, drop the subcommand wording. its just "Stores all commands registered to this command group."